SAKPaaS/src/app/components/location-panel/location-panel.component.scss

```
@@ -50,3 +49,42 @@ h1 {
 .sectionheading {
   margin-top: 30px;
 }
+
+.searchMinimized {
+
+}
+
+.minimized {
+  top: calc(100vh - 120px) !important;
+}
+
+@media (min-width: 600px) {
+  .minimize-icon {
+    display: none;
+  }
+}
+
+@media (max-width: 600px) {
+  .minimize-icon {
+
+  }
+
+  .minimize {
+    font-size: 25px;
+    display: flex;
+    justify-content: center;
+  }
+
+  .location-panel {
+    top: 30%;
```

```suggestion
    top: 30vh;
```

would propably make more sense.